File: 3. CursoPOO/PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMateral;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMateral){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMateral = $seatsMateral;
        
    }

    public function setPassenger($passenger) {
        if ($passenger == 6) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 6 pasajeros";
        }
    
    }
}

?>

File: 3. CursoPOO/Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMateral = []

    def __init__(self, license, driver, typeCarAccepted, seatsMateral):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMateral = seatsMateral

    def setPassenger(self, passegengerNum):
        super().setPassenger(passegengerNum)
        if passegengerNum >=6:
            self.__passegenger = int(passegengerNum)
        else:
            print("Necesita asignar 6 pasajeros")